Resources/Views/home.leaf

```diff
@@ -0,0 +1,87 @@
+#extend("base")
+
+#export("head") {
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="/css/explore.css">
+    <title>HomeKitty - Browse HomeKit Accessories</title>
+}
+
+
+#export("body") {
+    #embed("navigation")
+    <div class="container container-content">
+        <div class="row">
+            <!-- Side Menu -->
+            <div class="col-md-3">
+                <p class="lead">Search</p>
+                <div class="input-group" style="display: block;">
+                    <form action="/explore/search" style="display: flex;">
+                    <input type="text" class="form-control" placeholder="Search" name="term">
+                    <span class="input-group-btn" style="margin-right: 35px;">
+                        <button type="submit" class="btn btn-default" type="button"><span class="glyphicon glyphicon-search"></span></button>
+                    </span>
+                    </form>
+                </div>
+                <br>
+                <p class="lead">Categories</p>
+                <div class="list-group">
+                    #loop(categories, "category") {
+                    <a href="/explore?category=#(category.name)" class="list-group-item #if(category.selected) { selected }"><img src="#(category.image)" width="30px"> #(category.name)<span class="badge">#(category.accessoriesCount)</span></a>
+                    }
+                </div>
+            </div>
+
+            <!-- Content -->
+            <div class="col-md-9">
```

Comment:
Can we wrap this featured item div in a `#if()` to check if the featured item exists? In this way, in case we want to temporarily remove the accessory, we can just remove the featured product from the `Node` which is rendered and avoid to modify this `leaf` file every time 😄